Source/Scene/Model.js

```diff
@@ -2733,10 +2832,11 @@ define([
             }
         } else {
             createBuffers(model, context); // using glTF bufferViews
-            createPrograms(model, context);
             createSamplers(model, context);
             loadTexturesFromBufferViews(model);
             createTextures(model, context);
+
+            createPrograms(model, context);
```

Comment:
This was leftover from a previous iteration of changes and shouldn't have been changed. I'll put it back.